job/boot.sh

```diff
@@ -23,6 +23,15 @@ boot() {
   ret=$?
   trap before_exit EXIT
   [ "$ret" != 0 ] && return $ret;
+
+  exec_cmd $'echo "IS_CONTAINER=true" > <%= obj.jobInfoPath %>'
```

Comment:
- so lets not generate ANY scripts using the redirection operator. if we need something, add it as a template/bash script and then use envs or js injection to populate it but this should not be done anywhere. 
- same comment for the next line

Reply:
I'm not sure about how this should work. These are generated at runtime, so should we keep a copy of the template somewhere, make use of something like sed to replace it?

Reply:
- also, can we not inject the correct value of `IS_CONTAINER` based on whether the job is running as a container or not. this seems to be always injecting `true` in boot.sh